codemods/es-string-html-methods/index.js

```diff
@@ -0,0 +1,82 @@
+import jscodeshift from 'jscodeshift';
+import { removeImport } from '../shared.js';
+
+/**
+ * @typedef {import('../../types.js').Codemod} Codemod
+ * @typedef {import('../../types.js').CodemodOptions} CodemodOptions
+ */
+
+/**
+ * @param {CodemodOptions} [options]
+ * @returns {Codemod}
+ */
+export default function (options) {
+	return {
+		name: 'es-string-html-methods',
+		transform: ({ file }) => {
+			const j = jscodeshift;
+			const root = j(file.source);
+
+			const methods = [
+				'anchor',
+				'big',
+				'blink',
+				'bold',
+				'fixed',
+				'fontcolor',
+				'fontsize',
+				'italics',
+				'link',
+				'small',
+				'strike',
+				'sub',
+				'sup',
+			];
+
+			// Remove all imports
+			const methodIdentifiers = methods.reduce((acc, method) => {
+				//@ts-ignore
+				acc[method] = removeImport(
+					`es-string-html-methods/${method}`,
+					root,
+					j,
+				).identifier;
+				return acc;
+			}, {});
```

Comment:
you might be able to avoid the tsignore if you use Object.fromEntries, and .map()?

Reply:
Cool thanks! That's nicer.